taipy/gui/data/data_accessor.py

```diff
@@ -132,11 +133,21 @@ def _register(self, cls: t.Type[_DataAccessor]) -> None:
                 for cl in classes:
                     self.__access_4_type[cl] = inst  # type: ignore
 
+    def _unregister(self, cls: t.Type[_DataAccessor]) -> None:
+        """Unregisters a DataAccessor type."""
+        if cls in self.__access_4_type:
+            del self.__access_4_type[cls]
+
     def __get_instance(self, value: _TaipyData) -> _DataAccessor:  # type: ignore
         value = value.get() if isinstance(value, _TaipyData) else value
         access = self.__access_4_type.get(type(value))
         if access is None:
             if value is not None:
+                converted_value = type(self.__gui)._convert_unsupported_data(value)
```

Comment:
is the `type(self.__gui)` needed ?
I would think that `self.__gui.._convert_unsupported_data(value)` would work ?

Reply:
`_convert_unsupported_data()` is a class method.

Reply:
I get that but it can be called on the instance, no ?

Reply:
No it cannot. `<class>.method()` is different from `<instance>.method()`.